pkg/client/config.go

```diff
@@ -115,7 +115,7 @@ type Timeouts struct {
 	// PrivateTrafficManagerConnect is how long to wait for the initial port-forwards to the traffic-manager
 	PrivateTrafficManagerConnect time.Duration `json:"trafficManagerConnect,omitempty"`
 	// Helm is how long to wait for any helm operations.
```

Comment:
pls update this comment and remove the one on line 101 (`the exception is the helm timeout...`) since it no longer applies :)